app/models/bulkrax/csv_entry.rb

```diff
@@ -108,10 +108,11 @@ def build_mapping_metadata
 
     def build_object(object_key, value)
       data = hyrax_record.send(value['object'])
-      next if data.empty?
```

Comment:
this returns "syntaxerror: invalid next" because you can't call `next` outside a loop

Reply:
yes, I refactored at the very end to make rubocop happy. this should just be `return if data.empty?`

Reply:
yup! that's what I changed it to. 🙂